.github/icon_checklist.md

```diff
@@ -0,0 +1,24 @@
+Hey! Thanks for the contribution. While there is no reviewer, you can do a self-review. This will help to fix the reasons why the icons may not be suitable for Lawnicons. 
+
+### Canvas and sizes [→](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#canvas--sizes)
```

Comment:
Using `→` isn't really accessible. You could replace it with `(more info)` instead or simply make the whole heading a link.

This goes to all of the headers using `→`.

Reply:
They are specially made like this. It is necessary that they do not stand out too much relative to more relevant information for self-review. In terms of accessibility, it is enough that they are large and stand out in color.

Reply:
I'll try to see any other alternative solution soon-ish, but sure, this will work for now.

Reply:
I had the idea not to link to individual sections, but to link only to the guide. We can use it. For example, at the end, write "The Lawnicons guidelines".

Reply:
Sure thing. Here's an example:

```markdown
### More information
For a complete guide on the icon guidelines, visit the [Lawnicons Contributing Guidelines](CONTRIBUTING.md).
```

Reply:
I deleted it in the end, because the link to the full guidelines is in the PR description template.